.github/workflows/check-install.yml

```diff
@@ -0,0 +1,52 @@
+name: Check install
+
+# See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency.
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+on:
+  push:
+    paths-ignore:
+      - "doc/**"
+      - "**/README.md"
+      - "CONTRIBUTING.md"
+      - "changelog.d/**"
+      # only top level for these, because various test packages have them too
+      - "*/ChangeLog.md"
+      - "*/changelog.md"
+      - "release-notes/**"
+  pull_request:
+    paths-ignore:
```

Comment:
If you ignore these, you don't catch the `README.md` problem also fixed in this PR.

Reply:
If that's the case then wouldn't keeping this as a separate workflow be simpler?

Reply:
Simpler, but if Artem's annoyed about having lots of little checks then you may get pushed into it anyway, in which case the combined check needs to always run instead of using the docs hack.

Reply:
Maybe it's the other way around, and the previous, "sdist" workflow shouldn't have used these ignore-paths?

Reply:
`check-sdist` wants to verify that a build works by going from an `sdist` as the closest it can get to "installing from Hackage". It's specifically looking for ABI breaks, so doc-only changes make no sense for it.